Gemfile.lock

```diff
@@ -71,73 +71,73 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (2.0.0)
       rails (>= 7.0)
-    activejob (7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activejob (7.0.4)
+      activesupport (= 7.0.4)
       globalid (>= 0.3.6)
-    activemodel (7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activerecord (7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
-    activestorage (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    activemodel (7.0.4)
+      activesupport (= 7.0.4)
+    activerecord (7.0.4)
+      activemodel (= 7.0.4)
+      activesupport (= 7.0.4)
+    activestorage (7.0.4)
+      actionpack (= 7.0.4)
+      activejob (= 7.0.4)
+      activerecord (= 7.0.4)
+      activesupport (= 7.0.4)
       marcel (~> 1.0)
       mini_mime (>= 1.1.0)
-    activesupport (7.0.2.3)
+    activesupport (7.0.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
-    acts_as_tenant (0.5.1)
+    acts_as_tenant (0.5.2)
       rails (>= 5.2)
       request_store (>= 1.0.5)
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
-    airbrussh (1.4.0)
+    addressable (2.8.1)
+      public_suffix (>= 2.0.2, < 6.0)
+    airbrussh (1.4.1)
       sshkit (>= 1.6.1, != 1.7.0)
     annotate (3.2.0)
       activerecord (>= 3.2, < 8.0)
       rake (>= 10.4, < 14.0)
-    apipie-rails (0.7.0)
+    apipie-rails (0.8.2)
       actionpack (>= 5.0)
       activesupport (>= 5.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.573.0)
-    aws-sdk-core (3.130.0)
+    aws-partitions (1.632.0)
+    aws-sdk-core (3.150.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.525.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-kms (1.55.0)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-kms (1.58.0)
```

Comment:
### [aws-sdk-kms](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.55.0 → 1.58.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-sdk-kms/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-sdk-kms), [gem-diff](https://my.diffend.io/gems/aws-sdk-kms/1.55.0/1.58.0)_]





<details>
<summary>Commits</summary>

A change of **29** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/v1.55.0...v1.58.0).

These are the first 10 commits:
- (9170022) [Added configuration option for a custom SSL ca cert store.](https://github.com/aws/aws-sdk-ruby/commit/91700223be98d1e12f63a24ff260c021ebb8f0cb)
- (cc3d48d) [Amazon Kinesis Doc Update](https://github.com/aws/aws-sdk-ruby/commit/cc3d48da08728b2f1a8736c9930e0f419914bfa2)
- (7260668) [Amazon Elastic Transcoder Doc Update](https://github.com/aws/aws-sdk-ruby/commit/726066814aa197b36f5fc1194b29fdf3c1b6ce67)
- (fa83f35) [API Updates for Amazon RDS](https://github.com/aws/aws-sdk-ruby/commit/fa83f350e442889a4fc14d08b7e6301d78932ecb)
- (789f202) [API Updates for Amazon CloudSearch](https://github.com/aws/aws-sdk-ruby/commit/789f202b5356f7e8973eb150ec816ff9622f12e1)
- (77cd3e1) [Tag release v1.56.0](https://github.com/aws/aws-sdk-ruby/commit/77cd3e188ad38e934daf07916553959a53bb636b)
- (881a0f1) [Handling S3 errors when using v2 signatures in eu-central-1.](https://github.com/aws/aws-sdk-ruby/commit/881a0f1fc72f07e9c0acb87883aa260799fb07f5)
- (f400355) [Tag release v1.57.0](https://github.com/aws/aws-sdk-ruby/commit/f4003552bce1b6cbd5db8093928587d24e131818)
- (b8d6f7b) [Updated bundled endpoints.json file.](https://github.com/aws/aws-sdk-ruby/commit/b8d6f7b2141b77fd88a78fd44712b637c7646435)
- (be709f5) [Updated spec to contain new region name.](https://github.com/aws/aws-sdk-ruby/commit/be709f5adb55113dc818ed1fd66297cc951c463d)


</details>